### ISSUE TITLE
nvme?

### ISSUE THREAD

**Issue by dsrtusr88** [freemansoft/win10-storage-spaces#5]
Do you need to do anything for an NVME or is it smart enough to treat it differently from a SATA SSD?

**Comment by dsrtusr88** [comment]
I ran the script as-is.  In my case, I have a SATA SSD 2TB and an NVME SSD 1 TB.  It set the SATA SSD utilization capped at 1TB for the NVME, is there a way around this?

**Comment by freemansoft** [comment]
I don't have a system with a spare NVME drive so I don't have a way to test. 

You have two drives, one SATA 2TB and one NVME 1TB, correct?

**Comment by dsrtusr88** [comment]
that's correct, from your notes, it sounds like they're stuck at which ever drive is smallest?

**Comment by freemansoft** [comment]
Just saw that you are talking all SSDs.  The code exists to use an SSD as a cache for slower spinning disks.  I'm not sure there is any speed advantage using an NVME in front of SSDs.  

The code specifically picks out drives by type.  I didn't code for an NVME so it probably isn't going to work.  It assumes the SATA SSDs are the cache and other drives are the slow drives. It may be assuming your NVME is a slow drive so the max it could handle is 1TB.
